### ISSUE TITLE
Trying to fine-tune gfpgan

### ISSUE THREAD

**Issue by NetPranav** [TencentARC/GFPGAN#604]
I wanted to increase the accuracy and train gfpgan on a particular lighting dataset to increase the details and lighting scenario of image precision 
But I am new to this and got stuck as again and again the error occurs that Gfpganer has no parameter I tried to do the same with calling various like generator and discriminator but it just didn't work  can any one tell me like how to actually FINETUNE a GFPGAN with new dataset

**Comment by shubhamghimire** [comment]
You can not finetune the pth file of Ver1.3 and Ver1.4 of GFPGAN. Because these pth files are converted from another pth files using  the following script.
https://github.com/TencentARC/GFPGAN/blob/master/scripts/convert_gfpganv_to_clean.py

So you need to get the original pth file to finetune GFPGAN.
As far as I know the original pth files for Ver.1.3 and Ver.1.4 are not provided. But there are pth files for Ver.1.1 which you can finetune. You can download the generator weights and discriminator weights from below.

https://drive.google.com/drive/folders/17rLiFzcUMoQuhLnptDsKolegHWwJOnHu

**Comment by SreehariRajan** [comment]
Can you please provide the original pth file for  Ver.1.4 ?
